test/built-ins/Set/GetSetRecord/called-with-object.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2023 Anthony Frehner. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
```

Comment:
All of these should have the esid that is used in the proposal, as I understand it. I definitely don't look forward to coming back to all these later to add them.